### ISSUE TITLE
AttributeError: 'str' object has no attribute 'size'

### ISSUE THREAD

**Issue by c0deWiZz** [bjascob/amrlib#8]
Hello there!

I'm not sure if this is a noobish problem, but I am out of my wits!

After setting up the package according to the documentation, I did run the proposed minimal code:

```bash
import amrlib
stog = amrlib.load_stog_model()
graphs = stog.parse_sents(['This is a test of the system.', 'This is a second sentence.'],True)
for graph in graphs:
    print(graph)
```

However, I get the following error:

```bash
$ python3 stog_test.py 
Loading model /home/gandalf/.local/lib/python3.7/site-packages/amrlib/data/model_stog/model.pt
Traceback (most recent call last):                                                            
  File "stog_test.py", line 5, in <module>                                                    
    graphs = stog.parse_sents(['This is a test of the system.', 'This is a second sentence.'],True)                                                                                         
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/inference.py", line 65, in parse_sents                                                                    
    return self.parse_file_handle(sio_f, add_metadata)                                        
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/inference.py", line 93, in parse_file_handle                                                              
    res = self._parse_batch(batch)                                                            
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/inference.py", line 152, in _parse_batch
    beams = self.model.work(batch, self.beam_size, self.max_time_step)
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/modules/parser.py", line 81, in work
    data['pos'], data['ner'], data['word_char'], data['bert_token'], data['token_subword_index'])
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/modules/parser.py", line 62, in encode_step_with_bert
    bert_embed, _ = self.bert_encoder(bert_token, token_subword_index=token_subword_index)
  File "/home/gandalf/.local/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/bert_utils.py", line 60, in forward
    return self.average_pooling(encoded_layers, token_subword_index), pooled_output
  File "/home/gandalf/.local/lib/python3.7/site-packages/amrlib/models/parse_gsii/bert_utils.py", line 66, in average_pooling
    _, num_total_subwords, hidden_size = encoded_layers.size()
AttributeError: 'str' object has no attribute 'size'
```

I'm not sure if this is related to amrlib at all, `encoded_layers` here is a String with the value `"last_hidden_layer"`.

My pip freeze:

```bash
amrlib==0.2.1
asn1crypto==0.24.0
beautifulsoup4==4.7.1
blis==0.7.3
Brlapi==0.6.7
catalogue==1.0.0
certifi==2018.8.24
chardet==3.0.4
click==7.1.2
cryptography==2.6.1
cupshelpers==1.0
cymem==2.0.4
dataclasses==0.6
distro==1.3.0
distro-info==0.21
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.1/en_core_web_sm-2.3.1.tar.gz
entrypoints==0.3
evdev==1.1.2
filelock==3.0.12
future==0.18.2
galternatives==1.0.4
graphviz==0.15
html5lib==1.0.1
httplib2==0.11.3
idna==2.6
importlib-metadata==3.1.1
joblib==0.17.0
keyring==17.1.1
keyrings.alt==3.1.1
louis==3.8.0
lxml==4.3.2
meteo-qt==1.0.0
murmurhash==1.0.4
nltk==3.5
numpy==1.19.4
olefile==0.46
packaging==20.7
Penman==1.1.0
pexpect==4.6.0
Pillow==5.4.1
plac==1.1.3
preshed==3.0.4
pycairo==1.16.2
pycrypto==2.6.1
pycups==1.9.73
pycurl==7.43.0.2
PyGObject==3.30.4
pyparsing==2.4.7
PyQt5==5.15.2
PyQt5-sip==12.8.1
PySimpleSOAP==1.16.2
pysmbc==1.0.15.6
python-apt==1.8.4.1
python-debian==0.1.35
python-debianbts==2.8.2
python-magic==0.4.16
pyxattr==0.6.1
pyxdg==0.25
PyYAML==3.13
regex==2020.11.13
reportbug===7.5.3-deb10u1
reportlab==3.5.13
requests==2.21.0
sacremoses==0.0.43
SecretStorage==2.3.1
setproctitle==1.1.10
six==1.12.0
smatch==1.0.4
soupsieve==1.8
spacy==2.3.4
srsly==1.0.4
thinc==7.4.3
tokenizers==0.9.4
torch==1.7.0+cpu
torchaudio==0.7.0
torchvision==0.8.1+cpu
tqdm==4.54.0
transformers==4.0.0
typing-extensions==3.7.4.3
unattended-upgrades==0.1
Unidecode==1.1.1
urllib3==1.24.1
wasabi==0.8.0
webencodings==0.5.1
word2number==1.1
youtube-dl==2020.12.2
zipp==3.4.0
```

If you need further information, let me know, I would appreciate any help!

**Comment by bjascob** [comment]
I think the issue is the "transformers" package.  They released version 4.0.0 a few days ago and there are some breaking changes in that release.  I've tested the code with version 3.4.0 and now version 3.5.1 (latest 3.x release).  Can you try uninstalling version 4 and then do...
`pip3 install transformers==3.5.1` and let me know if that fixes the issue.  If so, I'll look into code changes to work with the new release.

**Comment by c0deWiZz** [comment]
Thank you so much, it's working now :). Looking forward to work with the package!
